### ISSUE TITLE
Youtube ads playing despite adblock being enabled

### ISSUE THREAD

**Issue by viasux** [bromite/bromite#2354]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md).
- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have updated Bromite to the latest version. The bug is reproducible on this latest version.
- [X] This is a bug report about the Bromite browser; not the website nor F-Droid nor anything else.

### Can the bug be reproduced with corresponding Chromium version?

No

### Bromite version

105.0.5195.41

### Device architecture

arm64

### Android version

12.1

### Device model

Google Pixel 4a5g

### Changed flags

no flags changed

### Is this bug about the SystemWebView?

No

### Is this bug happening in an incognito tab?

Yes

### Is this bug caused by the adblocker?

Yes

### Is this bug a crash?

No.

### Describe the bug

Youtube ads still play while adblock is enabled. I understand that it is possible to add custom adblock rules, however youtube is a very popular site and I believe ads should be blocked on it by default. Is this possibly because of something to do with manifest v3?

I had to remake this bug report because "Can the bug be reproduced with corresponding Chromium version?" was set to yes and it was immediately closed (of course its reproducible on chromium because chromium doesnt have an adblocker lol)

### Steps to reproduce the bug

1. Go to youtube.com
2. Watch a video
3. See ads

### Expected behavior

I expect ads to not show on youtube.com

### Screenshots

![image](https://user-images.githubusercontent.com/47131867/192371876-86e53fdf-8c87-4c58-b3ff-c3ec33696c77.png)

**Comment by csagan5** [comment]
> * I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).

I don't think you have read the FAQs.

**Comment by viasux** [comment]
> > * I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
> 
> I don't think you have read the FAQs.

I did read it, not sure what I'm missing, does YouTube use cosmetic filtering?

**Comment by csagan5** [comment]
I am referring to this question: https://github.com/bromite/bromite/blob/master/FAQ.md#some-sites-show-ads-how-can-i-fix-this

Please do some searches before opening new issues.
